environment-dev.yml

```diff
@@ -1,8 +1,9 @@
 channels:
     - conda-forge
 dependencies:
+    - pkg-config
     - fftw>=3.3.8
```

Comment:
Shouldn't we also add meson-python as a dependency as its need to build from source?